packages/android_intent_plus/lib/android_intent.dart

```diff
@@ -179,6 +179,20 @@ class AndroidIntent {
     );
   }
 
+  /// Sends intent as broadcast.
```

Comment:
This comment line is incorrect, should be:

```suggestion
  /// Starts intent as service.
```